server/controllers/wazuh-reporting.js

```diff
@@ -1895,6 +1896,20 @@ export class WazuhReportingCtrl {
         const isAgentConfig = tab === 'agentConfig';
         const isGroupConfig = tab === 'groupConfig';
 
+        // Generate the filename of report depeding on request parameters
+        const filename = tab === 'syscollector'
+          ? `wazuh-agent-inventory-${agentID}-${this.generateReportTimestamp()}.pdf`
```

Comment:
Here you are expecting the ID of the agent in `agentID`, but actually the react service sends the agent ID in the `isAgents` parameter. 

This is the request when clicking Agents / Inventory PDF report
![image](https://user-images.githubusercontent.com/9343732/183106192-052d5f9b-e5de-4b01-9ff7-4417d3120985.png)

![image](https://user-images.githubusercontent.com/9343732/183106289-6f27009e-9fb8-48b1-9520-95f649ca0ef3.png)

Reply:
Good catch! It is fixed now.